service/src/main/kotlin/fi/espoo/evaka/invoicing/service/InvoiceService.kt

```diff
@@ -73,6 +73,10 @@ class InvoiceService(
         val original = tx.getInvoice(uuid)
             ?: throw BadRequest("No original found for invoice with given ID ($uuid)")
 
+        if (invoice.rows.any { it.amount < 0 }) {
+            throw BadRequest("Invoice rows cannot have negative amounts")
+        }
```

Comment:
I don't think `0` really makes sense either.

Reply:
My thinking behind allowing `0` was that maybe they want to have "comment rows" or such 🤔

Reply:
Although then it would be possible to use `1` as amount and `0` as price, too.

Reply:
Fixed 👍